### ISSUE TITLE
Error with the Bot

### ISSUE THREAD

**Issue by kzarrabi24** [SwapnikKatkoori/sleeper-ff-bot#22]
I just added this bot to my discord server and each day I am getting a welcome message for 2020 and not 2021 and today on the 3rd day of adding the bot I am getting this error: "There was an error that occurred with the bot: 'NoneType' object is not iterable"

**Comment by ryeniner** [comment]
Yeah I added it back in July and get the same 3-4 errors every day. Was hoping it would start working this past Thursday with the season officially starting but I guess not.
